indico-prod/worker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9-bullseye
+FROM python:3.11-bullseye
```

Comment:
could even try `3.11-bookworm` (this is the current debian stable, bullseye is oldstable since about a month)